ethsigner/core/build.gradle

```diff
@@ -52,9 +52,15 @@ dependencies {
   testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
 
   integrationTestImplementation project(':ethsigner:signer:file-based')
+  integrationTestImplementation 'org.junit.jupiter:junit-jupiter-api'
   integrationTestImplementation 'io.rest-assured:rest-assured'
   integrationTestImplementation 'org.assertj:assertj-core'
-  integrationTestImplementation 'org.mock-server:mockserver-netty'
+  integrationTestImplementation('org.mock-server:mockserver-netty') {
+    // mock-server includes a dependency on junit 4 for a junit rule which we aren't using
+    exclude group: 'junit', module: 'junit'
+  }
```

Comment:
May be we can give Hoverfly a channce in future :). It integrates very well with JUnit5 and very powerful for mocking API/REST testing.

Reply:
Should quite trivial to swap mock-server out for something else if we want. Mock-server does have support for junit-5 actually, but I think the extent of the support is the equivalent of a rule which we don't need.